cameraview/src/main/java/com/otaliastudios/cameraview/CameraController.java

```diff
@@ -72,6 +72,7 @@
     protected Size mCaptureSize;
     protected Size mPreviewStreamSize;
     protected int mPreviewFormat;
+    protected long mAutoFocusResetDelayMillis = CameraView.DEFAULT_AUTOFOCUS_RESET_DELAY_MILLIS;
```

Comment:
You don't need to initialize this here, because now it will be done in CameraView constructor.